File: C/Ex32.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    int n1,n2,i,soma;
	printf ("Numero positivo menor que 100: ");
	scanf ("%i", &n1);
	
	while ((n1<0) || (n1>100))
	{
		printf ("Numero invalido, digite novamente: ");
    	scanf ("%i", &n1);
	}	
	
soma=0;
	for (i=1; i<=n1; i++)
	{
		printf ("%i\n",soma);		
		n2=(i*i)+1;
        soma=soma+n2;
	}	
return 0;
}

File: Python/Ex32.py
soma=0
n1=int(input("Numero positivo menor que 100: "))
while (n1<=0) or (n1>100):
    n1=(input("Numero invalido, digite novamente: "))
for i in range (1, n1+1, 1):
    n2=(i*i)+1
    soma=soma+n2
print(f"A soma dos {n1:.0f}"f" numeros em sequencia = {soma:.0f}") 

